### DISCUSSION TITLE
REMOTE CONNECTION

### DISCUSSION THREAD

**Discussion by SansK4r** [community/community#123494] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I AM UNABLE TO CONNECT THE CDESPACE

**Comment by Akash1134** [comment, marked as answer]
Hi @SansK4r ,

Please use our [guide to troubleshoot](https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-your-connection-to-github-codespaces)  your connection to GitHub Codespaces.

Thanks!

**Reply by SansK4r** [reply]
AND HOW I CAN CONNECT GITHUB CODESPACE WITH VSCODE IN WINDOWS
